src/dotnet-interactive/Commands/Quit.cs

```diff
@@ -0,0 +1,28 @@
+﻿// Copyright (c) .NET Foundation and contributors. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+
+using System;
+using System.Threading.Tasks;
+using Microsoft.DotNet.Interactive.Commands;
+
+namespace Microsoft.DotNet.Interactive.App.Commands
+{
+    public class Quit : KernelCommandBase
+    {
+        public Quit(Action processTermination = null, string targetKernelName = null): base(targetKernelName)
```

Comment:
```suggestion
        public Quit(Action onQuit = null, string targetKernelName = null): base(targetKernelName)
```

It's good to avoid using two different words that mean effectively the same thing.

Reply:
Done